### PR TITLE
add ripemd160 hash to KRYPTO module

### PR DESCRIPTION
**Pull request by dwightguth**
This is needed for the RIPEMD160 precompiled contract in the EVM semantics.

### CODE REVIEW

File: k-distribution/tests/builtins/ripemd160.crypto [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ripemd160DigestTest("")
+ripemd160DigestTest("foo")
```
Comment (msaxena2) [inline k-distribution/tests/builtins/ripemd160.crypto]: Is it possible to add the sources that w can refer to - like a link to a document with examples? (like in the other test files)

**Review by msaxena2 — COMMENTED** [review]
Looks good apart from the one minor comment. Thanks @dwightguth for fixing the documentation for the other functions.

**Comment by msaxena2** [comment]
Also, can you squash the two commits into one. Seems like the second commit isn't necessarily an entity on its own.

**Comment by dwightguth** [comment]
@msaxena2 please re-review

**Review by msaxena2 — APPROVED** [review]

**Pull request merged**